src/main/java/seedu/duke/parsermanager/ParseAddProperty.java

```diff
@@ -178,6 +238,15 @@ private void checkForPriceNumberFormat(String budget) throws InvalidPriceFormatE
         }
     }
 
+    private String checkForValidUnitType(String unitTypeLabel) throws  InvalidUnitTypeLabelException {
```

Comment:
Note the extra space between `throws` and `InvalidUnitTypeLabelException`. But it's probably a small issue since CI doesn't complain.